README.md

```diff
@@ -16,28 +16,146 @@ This simple implementation stores bubbles as directories on the server's local f
 
 ## Dependencies
 
-Requires access to an EVM-compatible blockchain via either a Web3 api service such as [Infura](https://www.infura.io/), a local blockchain node or a local [Ganache](https://trufflesuite.com/ganache/) instance.  See Configuration.
+Requires access to an EVM-compatible blockchain via either a Web3 api service such as [Infura](https://www.infura.io/), a local blockchain node or a local [Ganache](https://trufflesuite.com/ganache/) instance. Setup your Infura api or local Ganache.
 
 
-## Configuration
+# Bubble Server Configuration
 
 Use `config.json` to specify the server port and each chain's chain id, blockchain api, endpoint and bubble storage directory.
 
-## Usage
+This guide explains the settings available in `config.json` for configuring the Bubble server with multiple blockchain networks.
+
+## Configuration Options
+
+### Server Port (`serverPort`)
+The main server port that your Bubble server will listen on, such as `3000`.
+
+### Chains (`chains`)
+Define each blockchain network with the following options:
+
+- **name**: A friendly name for each blockchain network (e.g., "Ethereum Mainnet", "Goerli Testnet", "Ganache Local").
+- **chainId**: The unique chain ID for the blockchain network (e.g., `1` for Ethereum mainnet, `5` for Goerli testnet, `1337` for a local Ganache network).
+- **web3Url**: The URL for the Web3 provider:
+  - For Infura or another Web3 API provider, use the respective URL with an API key (e.g., `https://mainnet.infura.io/v3/YOUR_INFURA_PROJECT_ID`).
+  - For a local Ethereum node, use its RPC endpoint (e.g., `http://localhost:8545`).
+  - For a local Ganache instance, use `http://127.0.0.1:8545`.
+- **endpoint**: The endpoint path for this blockchain, used for routing requests (e.g., `/eth-mainnet`).
+- **storageDirectory**: The path for local storage of bubbles on this blockchain network.
```

Comment:
`rootPath` not `storageDirectory` (although storageDirectory is more meaningful!)